### ISSUE TITLE
Infinity power

### ISSUE THREAD

**Issue by navaroneee** [MachineMuse/MachineMusePowersuits#786]
1.7.10 support ends? I tested the mod on a server, and it happens that the power never ends, it arrives at 20k and it stays the same always, and I can still use all the resources, besides it never heats up, offline it works ok

sorry my bad english :)

**Comment by lehjr** [comment]
yes, support has ended. And even if it had not, without telling me the version of the mod, plus the version of Numina, plus the version of Forge, there is nothing I would be able to do anyway. Remember, I can't see your screen, so I have no way of knowing what versions you are using.

**Comment by navaroneee** [comment]
Numina-0.4.1.104
ModularPowersuits-1.7.10-0.11.1.114
Forge 1.7.10 - 10.13.4.1614

Yes, i know, sorry not posted before

**Comment by lehjr** [comment]
There is a newer version of Numina

**Comment by navaroneee** [comment]
It's beta version right? I've tried too, but no sucess
